src/resources/app/components/AnnouncementCard.vue

```diff
@@ -29,12 +29,61 @@
         :id="comment.id"
       />
     </div>
+    <div>
+      <v-divider></v-divider>
+      <!--comments form-->
```

Comment:
nit: instead of adding a comment, why don't we refactor it to a new component

Reply:
Maybe in the next sprint, thanks